Dockerfile

```diff
@@ -44,7 +46,6 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && rm -rf /var/lib/apt/lists/* /root/.m2
 
 # Base part of dockerfile should not be modified not to rebuild image for 1 hour
-ENV JOLOKIA_VERSION="1.3.7"
```

Comment:
What is the purpose of moving this variable up? The install here takes time and reordering commands will cause longer rebuild